modules/Layout/Header/SearchWidget/components/SearchBar.tsx

```diff
@@ -5,7 +5,6 @@ import { connectSearchBox } from 'react-instantsearch-dom';
 import { FormattedMessage, useIntl } from 'react-intl';
 
 import { Button, FormInput } from '@/components';
-import { IconEnterKey } from '@/icons';
```

Comment:
Please remove the icon completely (the svg file and the definition) if it's not used anymore.